qualtran/bloqs/hamiltonian_simulation/hamiltonian_simulation_by_gqsp.py

```diff
@@ -197,9 +213,8 @@ def _symbolic_hamsim_by_gqsp() -> HamiltonianSimulationByGQSP:
 
     from qualtran.bloqs.hubbard_model import get_walk_operator_for_hubbard_model
 
-    walk_op = get_walk_operator_for_hubbard_model(2, 2, 1, 1)
-
-    t, inv_eps = sympy.symbols("t N")
+    tau, t, inv_eps = sympy.symbols(r"\tau t \epsilon^{-1}", positive=True)
```

Comment:
would epsilon be a better variable here since you're just passing in 1 / inv_eps anyway?

Reply:
Ah this is my failed attempt to fix sympy messing up `log` terms inside `Order`.

```py
from sympy import Symbol, log
from qualtran.resource_counting import big_O

x = Symbol('x', positive=True)
print(big_O(log(x)))
print(big_O(log(1 / x)))
print(big_O(log(log(x))))
```
outputs
```
O(log(1/x), (x, oo))
O(log(1/x), (x, oo))
O(log(-log(1/x)), (x, oo))
```

I'll keep track of this in an issue for now, and simplify the code to use `epsilon`.

Reply:
Ah no, I remember now, `sympy.Order` only supports multivariate limits to the same point, so I picked the symbol `1/eps -> inf` (instead of `eps -> 0`)